### PR TITLE
Add Instructions for Creating and Removing Proxy via Apps.Moonbeam.Network

### PR DESCRIPTION
**Pull request by themacexpert**
### Description

Apps.Moonbeam.Network added support for creating and removing proxies via UI. This PR supplements our existing proxy documentation with additional guidance for apps.moonbeam.network.

### Checklist

- [x] I have added a label to this PR 🏷️
- [x] I have run my changes through Grammarly
- [ ] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [ ] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects
- [ ] If pages have been moved around, I have run the `move-pages.py` script to move the pages and update the image paths on the chinese repo
    - [ ] After the script has been run, I have created an additional PR in `moonbeam-docs-cn`
- [ ] If images have been added, I have run the `compress-images.py` script to compress the images.
- [ ] If variables (in variables.yml) need to be updated (such as a name change), I have updated the `moonbeam-docs-cn` repo to use the new variables
- [ ] If this page requires a disclaimer, I have added one

### Corresponding PRs

Please link to any corresponding PRs here.

### After Translation Requirements

- [ ] Will need to create PR in `moonbeam-docs` repo to remove images
- [ ] Will need to create PR in `moonbeam-docs` repo to remove variables
- [ ] Will need to create PR in `moonbeam-mkdocs` repo to add redirects for Chinese site
- [ ] No additional PRs are required after the translations are done

#### Items to be Updated

Please list any of the items that will need to be added or deleted after the translations are done here.

### CODE REVIEW

File: tokens/manage/proxy-accounts.md [hunk 1]
```diff
@@ -137,17 +166,17 @@ To check your proxy accounts from the [**Chain state** page](https://polkadot.js
...
 
-![Verify your proxy accounts via the Extrinsics page of Polkadot.js Apps.](/images/tokens/manage/proxy-accounts/proxies-5.webp)
+![Verify your proxy accounts via the Extrinsics page of Polkadot.js Apps.](/images/tokens/manage/proxy-accounts/proxies-8.webp)
 
 The result will appear on the page showing you information about all of your proxies, including the delegate/proxy account address, the proxy type, the delay period if one was specified, and the total bond amount for all of your proxies in Wei.
 
 As previously mentioned, you can also check your proxy accounts from the **Accounts** page. To do so, you can navigate to the **Accounts** page and there should be a Proxy symbol next to the primary account. Hover over the icon and click on **Manage proxies** to review your proxies.
 
-![Hover over the proxy icon to manage your proxies via the Accounts page of Polkadot.js Apps.](/images/tokens/manage/proxy-accounts/proxies-6.webp)
+![Hover over the proxy icon to manage your proxies via the Accounts page of Polkadot.js Apps.](/images/tokens/manage/proxy-accounts/proxies-9.webp)
```
Comment (dawnkelly09) [inline tokens/manage/proxy-accounts.md:175]: Your file is actually named `proxies-9o.webp` so this image isn't displaying

**Review by dawnkelly09 — CHANGES_REQUESTED** [review]
Nice work, Kevin. One little edit to fix a broken image but, otherwise looks great! This is a nice alternative for devs who want to set up and verify these proxies and you explained it well.

File: tokens/manage/proxy-accounts.md (hunk 1 above)
Comment (themacexpert) [reply tokens/manage/proxy-accounts.md:175]: Thx Dawn good catch!

**Review by dawnkelly09 — DISMISSED** [review]
LGTM. Thanks, Kevin!

File: tokens/manage/proxy-accounts.md [hunk 2]
```diff
@@ -84,6 +84,23 @@ For the purposes of this guide, you will be setting up a proxy account using the
 
 ## Creating a Proxy Account {: #creating-a-proxy-account }
 
+
+### Via Apps.Moonbeam.Network {: #create-via-apps-moonbeam-network }
```
Comment (eshaben) [inline tokens/manage/proxy-accounts.md:88]:
```suggestion
### Via the Moonbeam DApp {: #create-via-apps-moonbeam-network }
```

File: tokens/manage/proxy-accounts.md [hunk 3]
```diff
@@ -107,27 +124,39 @@ As previously mentioned, you can also create a proxy from the **Accounts** page.
...
 3. Click **Submit** and sign the transaction
 
-![Add the details of the proxy account, including the proxy account and type.](/images/tokens/manage/proxy-accounts/proxies-4.webp)
+![Add the details of the proxy account, including the proxy account and type.](/images/tokens/manage/proxy-accounts/proxies-5.webp)
 
 In the next section, you will learn how to verify that your proxy account was set up successfully.
 
 ## Verifying your Proxy Account {: #verifying-your-proxy-account }
 
+### Via Apps.Moonbeam.Network {: #verify-via-apps-moonbeam-network }
```
Comment (eshaben) [inline tokens/manage/proxy-accounts.md:148]:
```suggestion
### Via the Moonbeam DApp {: #verify-via-apps-moonbeam-network }
```

File: tokens/manage/proxy-accounts.md [hunk 4]
```diff
@@ -107,27 +124,39 @@ As previously mentioned, you can also create a proxy from the **Accounts** page.
...
+
+When connected to [apps.moonbeam.network](https://apps.moonbeam.network/moonbeam/proxy){target=\_blank} with your primary account, you can see a list of accounts with proxy control over your connected primary account in the **Your Proxies** section.
```
Comment (eshaben) [inline tokens/manage/proxy-accounts.md:150]:
```suggestion
When connected to [the Moonbeam dApp](https://apps.moonbeam.network/moonbeam/proxy){target=\_blank} with your primary account, you can see a list of accounts with proxy control over your connected primary account in the **Your Proxies** section.
```

Using the URL as the link text will result in some of my reports returning issues with descriptive link text. It's an accessibility thing

File: tokens/manage/proxy-accounts.md [hunk 5]
```diff
@@ -84,6 +84,23 @@ For the purposes of this guide, you will be setting up a proxy account using the
 
 ## Creating a Proxy Account {: #creating-a-proxy-account }
 
+
+### Via Apps.Moonbeam.Network {: #create-via-apps-moonbeam-network }
+
+It's easy to create a proxy account on [apps.moonbeam.network](https://apps.moonbeam.network/moonbeam/proxy){target=\_blank}. To do so, take the following steps:
```
Comment (eshaben) [inline tokens/manage/proxy-accounts.md:90]: Using the URL as the link text will result in some of my reports returning issues with descriptive link text. It's an accessibility thing

```suggestion
It's easy to create a proxy account on [the Moonbeam dApp](https://apps.moonbeam.network/moonbeam/proxy){target=\_blank}. To do so, take the following steps:
```

File: tokens/manage/proxy-accounts.md [hunk 6]
```diff
@@ -107,27 +124,39 @@ As previously mentioned, you can also create a proxy from the **Accounts** page.
...
 
 ## Verifying your Proxy Account {: #verifying-your-proxy-account }
 
+### Via Apps.Moonbeam.Network {: #verify-via-apps-moonbeam-network }
+
+When connected to [apps.moonbeam.network](https://apps.moonbeam.network/moonbeam/proxy){target=\_blank} with your primary account, you can see a list of accounts with proxy control over your connected primary account in the **Your Proxies** section.
+
+![Verify proxy created via apps.moonbeam.network](/images/tokens/manage/proxy-accounts/proxies-6.webp)
+
+Alternatively, by connecting the proxy account to [apps.moonbeam.network](https://apps.moonbeam.network/moonbeam/proxy){target=\_blank}, you can see a list of accounts over which the connected account has proxy control in the **Proxied accounts to you** section.
```
Comment (eshaben) [inline tokens/manage/proxy-accounts.md:154]:
```suggestion
Alternatively, by connecting the proxy account to [the Moonbeam dApp](https://apps.moonbeam.network/moonbeam/proxy){target=\_blank}, you can see a list of accounts over which the connected account has proxy control in the **Proxied accounts to you** section.
```

File: tokens/manage/proxy-accounts.md [hunk 7]
```diff
@@ -175,6 +204,14 @@ That's it! You've successfully executed a transaction using a proxy account on b
 
 ## Removing a Proxy Account {: #removing-a-proxy-account }
 
+### Via Apps.Moonbeam.Network {: #remove-via-apps-moonbeam-network }
+
+To remove a proxy account, connect your primary account to [apps.moonbeam.network](https://apps.moonbeam.network/moonbeam/proxy){target=\_blank} and press **Remove** next to the proxy account that you want to remove. Alternatively, you can remove all proxy accounts of the primary account with **Remove All Proxies**. In either case, you must confirm the transaction in your wallet.
```
Comment (eshaben) [inline tokens/manage/proxy-accounts.md:209]:
```suggestion
To remove a proxy account, connect your primary account to [the Moonbeam dApp](https://apps.moonbeam.network/moonbeam/proxy){target=\_blank} and press **Remove** next to the proxy account that you want to remove. Alternatively, you can remove all proxy accounts of the primary account with **Remove All Proxies**. In either case, you must confirm the transaction in your wallet. 
```

File: tokens/manage/proxy-accounts.md [hunk 8]
```diff
@@ -175,6 +204,14 @@ That's it! You've successfully executed a transaction using a proxy account on b
 
 ## Removing a Proxy Account {: #removing-a-proxy-account }
 
+### Via Apps.Moonbeam.Network {: #remove-via-apps-moonbeam-network }
```
Comment (eshaben) [inline tokens/manage/proxy-accounts.md:207]:
```suggestion
### Via the Moonbeam DApp {: #remove-via-apps-moonbeam-network }
```

**Review by eshaben — COMMENTED** [review]
Thanks Kevin! Just some minor stuff

**Review by eshaben — APPROVED** [review]
LGTM! 🚀

**Pull request merged**